common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_common.yaml

```diff
@@ -579,7 +579,7 @@ sBackhaulParams:
   backhaul_is_wireless: uint8_t 
   backhaul_iface_type: uint8_t 
   is_backhaul_manager: uint8_t 
-  gw_manage_type: uint8_t 
```

Comment:
Removal of this unused field should be made explicit in the commit message.

Reply:
Oops I forgot to update the commit message, mergify beat me to it